dbt/include/bigquery/macros/materializations/table.sql

```diff
@@ -49,12 +49,19 @@
 from pyspark.sql import SparkSession
 {%- set raw_partition_by = config.get('partition_by', none) -%}
 {%- set raw_cluster_by = config.get('cluster_by', none) -%}
+{%- set enable_list_inference = config.get('enable_list_inference', true) -%}
```

Comment:
Since this spawned from a regression, setting the default to `true` effectively mitigates the regression.